x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/queries/InterceptedInferenceKnnVectorQueryBuilderTests.java

```diff
@@ -308,6 +309,10 @@ public void testCoordinatorNodeRewrite_GivenKnnQueryWithSemanticFilters_ShouldIn
         }
     }
 
+    public void testCcsSerializationWithMinimizeRoundTripsFalse() throws Exception {
```

Comment:
Maybe make this an abstract method in `AbstractInterceptedInferenceQueryBuilderTestCase` to force any future tests to override?